_posts/2017-05-08-gsoc-2017-students.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: Google Summer of Code 2017 Students
+category: activism
+author: Genadi Samokovarov
+published: true
+date: 2017-05-08 18:00:00 -02:00
+---
+
+We super are excited to announce our accepted Google Summer of Code students
```

Comment:
I guess it's the other way around: "We are super excited to ...". Nice post so far ! 👍